### DISCUSSION TITLE
CodeIgniter 4

### DISCUSSION THREAD

**Discussion by MGatner** [deployphp/deployer#2191] [General]
CodeIgniter 3 is not recommended for new projects. I just discovered this tool from your Reddit post on r/PHP, but if I can help with a CodeIgniter 4 recipe I highly recommend offering it.

**Comment by antonmedv** [comment]
Hi, thanks! We have this https://github.com/deployphp/deployer/blob/master/recipe/codeigniter.php
But I don’t know how many people are using it and if you can create PR for CI4 it will be awesome!
